### PR TITLE
Upgrading storybook to 6.5

### PR DESCRIPTION
**Pull request by georgewrmarshall**
### Description
Upgrading storybook from `^6.4.18` to `^6.5.10` because it [includes a bunch of goodies](https://storybook.js.org/blog/storybook-6-5/#webpack-5-lazy-compilation-for-faster-startup).

### Manual testing steps
- Check out the storybook build from this PR or pull this branch and run `yarn setup && yarn storybook`
- Check that stories and docs still work as usual

### References
https://storybook.js.org/blog/storybook-6-5/#webpack-5-lazy-compilation-for-faster-startup

Screen Shot 2022-08-05 at 10 46 33 AM

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -18,5 +18,10 @@ module.exports = {
...
     },
   ],
-  ignorePatterns: ['!.eslintrc.js', '!.prettierrc.js', 'dist/'],
+  ignorePatterns: [
+    '!.eslintrc.js',
+    '!.prettierrc.js',
+    'dist/',
+    'storybook-static/',
+  ],
 };
```
Comment (georgewrmarshall) [inline .eslintrc.js:27]: Needed to add storybook static folder otherwise linting was failing

File: package.json [hunk 2]
```diff
@@ -60,7 +60,7 @@
     "eslint-plugin-jsdoc": "^36.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-storybook": "^0.5.6",
+    "eslint-plugin-storybook": "^0.6.3",
     "jest": "^26.4.2",
     "node-fetch": "^2.6.0",
     "prettier": "^2.2.1",
```
Comment (georgewrmarshall) [inline package.json:66]: Updating packages and using prettier to change scripts to alphabetical order

File: .eslintrc.js [hunk 3]
```diff
@@ -18,5 +18,10 @@ module.exports = {
       extends: ['@metamask/eslint-config-jest'],
     },
   ],
-  ignorePatterns: ['!.eslintrc.js', '!.prettierrc.js', 'dist/'],
+  ignorePatterns: [
+    '!.eslintrc.js',
+    '!.prettierrc.js',
+    'dist/',
+    'storybook-static/',
```
Comment (georgewrmarshall) [inline .eslintrc.js:25]: Adding storybook-static/ directory to ignore linting

**Review by Cal-L — APPROVED** [review]
LGTM!

**Pull request merged**